core/container/src/main/resources/modules/org/jboss/weld/se/main/module.xml

```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module xmlns="urn:jboss:module:1.5" name="org.jboss.weld.se">
   <resources>
-    <!--<resource-root path="weld-se-core-${version.weld}.jar"/>-->
-    <artifact name="org.jboss.weld.environment:weld-environment-common:${version.weld}"/>
-    <artifact name="org.jboss.weld:weld-api:${version.weld-api}"/>
-    <artifact name="org.jboss.weld:weld-spi:${version.weld-api}"/>
-    <artifact name="org.jboss.weld:weld-core-impl:${version.weld}"/>
-    <artifact name="org.jboss.weld.se:weld-se-core:${version.weld}"/>
+    <artifact name="org.jboss.weld.environment:weld-environment-common:${version.weld-se}"/>
+    <artifact name="org.jboss.weld:weld-api:${org.jboss.weld:weld-api:jar.version}"/>
```

Comment:
` version.weld-se` is defined in swarm root pom. `org.jboss.weld:weld-api:jar.version` and `org.jboss.weld:weld-core-impl:jar.version` are set by [dependencyversion-maven-plugin](https://github.com/wildfly-swarm/wildfly-swarm/pull/564/files#diff-1d9cc2c72a3cdd14f37073b9bd158ddaR41) and correspond to  https://github.com/wildfly/wildfly/blob/master/pom.xml#L200 (when importing a pom in <dependencyManagement> only dependencies are considered, not properties, so we need to take the current version of the weld artifacts).